protocol_rfcs/managed-commits.md

```diff
@@ -0,0 +1,345 @@
+# Managed Commits
+**Associated Github issue for discussions: https://github.com/delta-io/delta/issues/2598**
+
+This RFC proposes a new table feature `managedCommits` which changes the way Delta Lake manages commits.
+
+Today’s Delta commit protocol relies on the filesystem to provide commit atomicity. This feature request is to allow delta tables which gets commit atomicity using an external commit-owner and not
+the filesystem (s3, abfs etc). This allows us to deal with various limitations of delta:
```

Comment:
nit: s/delta/Delta/g for cases that refer to the table format (as opposed to "delta files" which are commit .json)